toolkit/pallets/address-associations/README.md

```diff
@@ -0,0 +1,84 @@
+# Address Associations Pallet
+
+## Overview
+
+The Address Associations pallet provides functionality to associate mainchain (i.e., Cardano) stake public keys with partner chain addresses. This forms a critical link between the main chain and the partner chain, enabling cross-chain identity verification and operations.
+
+## Purpose
+
+This pallet serves several purposes:
+- Establishes verifiable links between mainchain identities and partner chain addresses
+- Enables cross-chain validation of key ownership through cryptographic signatures
+- Provides a foundation for permission-based operations that require mainchain identity verification
+- Supports cross-chain governance and validator selection processes
```

Comment:
This is not really true. Validator selection is based on "registrations" (they happen on Cardano). Deliberately this pallet is name "associations" to prevent confusion with Stake Pool Operator registrations.

One concrete use-case it supports is delegators (entities that delegate their stake) rewards (hence PC address is "target" of the association).